src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Fundamentally (in my eyes, at least): Are ```.preventDefault()```, ```.stopPropagation()```, and ```.stopImmediatePropagation()``` defining properties of events? Are we as jQuery specifying that they are? If so, then everywhere an event appears it must be a well-formed event.

Equally fundamentally: Must every event.originalEvent. ... .originalEvent chain end in a native event? Currently, the answer would seem to be "no", since such a chain, constructed on top of a simulated event, is rooted in an empty object.